openapi/kas-fleet-manager.yaml

```diff
@@ -1563,6 +1563,10 @@ components:
           description: billing model to use
           type: string
           nullable: true
+        cluster_id:
+          description: enterprise OSD cluster ID to be used for kafka creation
+          type: string
+          nullable: true
```

Comment:
there is a ask to also return the cluster_id info for enterprise kafkas. 

The UI/CLI and other API consumer will base their logic off it.